packages/@aws-cdk/aws-codebuild/lib/source.ts

```diff
@@ -418,11 +448,11 @@ export class FilterGroup {
     if (this.actions.has(EventAction.PUSH)) {
       throw new Error('A base reference condition cannot be added if a Group contains a PUSH event action');
     }
-    return this.addFilter('BASE_REF', refName, include);
+    return this.addFilter(WebhookFilterTypes.BASE_REF, refName, include);
   }
 
   private addFilePathFilter(pattern: string, include: boolean): FilterGroup {
-    return this.addFilter(FILE_PATH_WEBHOOK_COND, pattern, include);
+    return this.addFilter(WebhookFilterTypes.FILE_PATH, pattern, include);
   }
 
   private addFilter(type: string, pattern: string, include: boolean) {
```

Comment:
Should this change the type to `(type: WebhookFilterTypes, pattern: string, include: boolean)`?

Reply:
Great point! I'm not very used to typed languages but that seems correct. Changes incoming soon